modules/monitoring-accessing-the-alerting-ui.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-alerts.adoc
+
+[id="accessing_the_alerting_ui_{context}"]
+= Accessing the Alerting UI in the Administrator and Developer perspectives
+
+The Alerting UI is accessible through the Administrator perspective and the Developer perspective in the {product-title} web console.
+
+* In the *Administrator perspective*, select *Monitoring* -> *Alerting*. The three main pages in the Alerting UI in this perspective are the *Alerts*, *Silences*, and *Alerting Rules* pages.
+
+//Next to the title of each of these pages is a link to the Alertmanager interface.
+
+* In the *Developer perspective*, select *Monitoring* -> *<project_name>* -> *Alerts*. In this perspective alerts, silences and alerting rules are all managed from the *Alerts* page. The results shown in the *Alerts* page are specific to the selected project.
+
+[NOTE]
+====
+In the Developer perspective, you can select from core {product-title} and user-defined projects that you have access to in the *Project:* drop-down menu. However, alerts, silences, and alerting rules relating to core {product-title} projects will not be displayed if you do not have `cluster-admin` privileges.
```

Comment:
IBM SG suggests to just say `*Project:* list` for a down-down menu/list, but... we have a whole lot of "drop-down menu" and "drop-down list" already throughout our docs. So probably not critical.

Reply:
Thank you for this suggestion, I think 'list' sounds much better. I have changed all instances of 'drop-down menu' to list throughout the PR.